src/store/ducks/plans.ts

```diff
@@ -466,6 +466,32 @@ export function getPlansArray(
   );
 }
 
+/** getFilteredPlansArray - get an array of Plans by intervention type
+ * That are also filtered by location and location hierarchy
+ * @param {Partial<Store>} state - the redux store
+ * @param {InterventionType} intervention - the intervention type
+ * @param {string[]} status - the plan status
+ * @param {string} reason - the plan reason
+ * @param {string} location - the location to be filtered in
+ * @param {number} locationHierarchy - the location hierarchy to look at
+ */
+export function getFilteredPlansArray(
```

Comment:
This function is very similar to the `getplansArray`. Probably modifying the existing used selector would retain the current level of dryness.